### PR TITLE
Added InvasionZ

### PR DESCRIPTION
**Pull request by AucuneInspiexe**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/invasionz/manifest.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  "server_name": "invasionz",
+  "nice_name": "InvasionZ",
+  "direct_ip": "play.invasionz.net",
+  "server_wildcards": [
+    "%.invasionz.net",
```
Comment (DoJapHD) [inline minecraft_servers/invasionz/manifest.json:6]: Not a wildcard
https://i.imgur.com/mXstph9.png

File: minecraft_servers/invasionz/manifest.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    "%.lunamc.fr"
```
Comment (DoJapHD) [inline minecraft_servers/invasionz/manifest.json:7]: Not a wildcard
https://i.imgur.com/oatF9xb.png

**Review by DoJapHD — CHANGES_REQUESTED** [review]

**Comment by DoJapHD** [comment]
Server is in maintenance mode; reopen PR once server is publicly available

**Comment by AucuneInspiexe** [comment]
Hi, when you replied it was just our MOTD that wasn't updating, but it's our fault. Now it's actually in maintenance and should be up in 6H, for the wildcards, we actually just have play.lunarmc.fr and play.invasionz.net, how should i make it so that we know those two ips instead of wildcards ? Should i just put play.invasionz.net and play.lunamc.net instead of %.invasionz.net ect ?

**Pull request closed without merging**